src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -76,212 +125,233 @@ limitations under the License.
       <description>ACLouverPosition</description>
-      <access op="read" privilege="view"/>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute side="server" code="0x0046" define="AC_COIL_TEMPERATURE" type="int16s" min="-27315" max="0x7FFF" writable="false" default="0x8000" optional="true" isNullable="true">ACCoilTemperature</attribute>
-    <attribute side="server" code="0x0047" define="AC_CAPACITY_FORMAT" type="enum8" min="0x00" max="0xFF" writable="true" default="0x00" optional="true">
+    <attribute side="server" code="0x0046" define="AC_COIL_TEMPERATURE" type="temperature" writable="false" default="0x8000" optional="true" isNullable="true">ACCoilTemperature</attribute>
+    <attribute side="server" code="0x0047" define="AC_CAPACITY_FORMAT" type="ACCapacityFormatEnum" min="0x0" max="0x0" writable="true" default="0" optional="true">
       <description>ACCapacityformat</description>
-      <access op="read" privilege="view"/>
       <access op="write" privilege="manage"/>
     </attribute>
     <command source="client" code="0x00" name="SetpointRaiseLower" optional="false">
       <description>
         Command description for SetpointRaiseLower
       </description>
-      <arg name="Mode" type="SetpointAdjustMode"/>
+      <arg name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
       <arg name="Amount" type="int8s"/>
     </command>
     <command source="client" code="0x01" name="SetWeeklySchedule" optional="true">
       <description>
         Command description for SetWeeklySchedule
       </description>
-      <arg name="NumberOfTransitionsForSequence" type="int8u"/>
-      <arg name="DayOfWeekForSequence" type="DayOfWeek"/>
-      <arg name="ModeForSequence" type="ModeForSequence"/>
-      <arg name="Transitions" type="ThermostatScheduleTransition" array="true"/>
       <access op="invoke" privilege="manage"/>
+      <arg name="NumberOfTransitionsForSequence" type="int8u"/>
+      <arg name="DayOfWeekforSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
+      <arg name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
+      <arg name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
     </command>
     <command source="client" code="0x02" name="GetWeeklySchedule" response="GetWeeklyScheduleResponse" optional="true">
       <description>
         Command description for GetWeeklySchedule
       </description>
-      <arg name="DaysToReturn" type="DayOfWeek"/>
-      <arg name="ModeToReturn" type="ModeForSequence"/>
+      <arg name="DaysToReturn" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
+      <arg name="ModeToReturn" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
     </command>
-    <command source="client" code="0x03" name="ClearWeeklySchedule" optional="true">
+    <command source="client" code="0x03" name="ClearWeeklySchedule" optional="true" disableDefaultResponse="true">
```

Comment:
The spec is buggy here.  It should have Y, not N, for the response to this command.  It even has prose talking about what status is sent in response! Please undo this change and get the spec fixed.

Reply:
@hasty can you make sure the spec gets the proper update?